src/sdltiles.cpp

```diff
@@ -2484,10 +2517,13 @@ void sfx::play_ambient_variant_sound( const std::string &id, const std::string &
         dbg( D_ERROR ) << "Failed to play sound effect: " << Mix_GetError();
     }
 }
+
 #endif
 
 void load_soundset() {
 #ifdef SDL_SOUND
+    unique_chunks.clear();
```

Comment:
Don't we only need the one `unique_chunks.clear()` at the end?

Reply:
Yes, will remove. I think I put that there then decided to put it at the end because it made more sense, then forgot to remove the first.